docs/architecture/edge/edgehub.md

```diff
@@ -13,7 +13,7 @@ It forwards the messages received from the cloud to the corresponding module at
 The main functions performed by edgehub are :-
 
 - Keep Alive
-- Publish Client Info
+- Publish Connection Info
```

Comment:
Based on the [code](https://github.com/kubeedge/kubeedge/blob/5b696a2d4b1ccd0edcbd9ce1a9c5bd58ababe683/edge/pkg/edgehub/process.go#L199), I think it should be `Publish Connection Info` , `Publish Clinet Info` is a wrong description